pkg/kine/prepared/db.go

```diff
@@ -22,15 +35,27 @@ func New(db *sql.DB) *DB {
 
 func (db *DB) Underlying() *sql.DB { return db.underlying }
 
-func (db *DB) ExecContext(ctx context.Context, query string, args ...any) (sql.Result, error) {
+func (db *DB) ExecContext(ctx context.Context, query string, args ...any) (result sql.Result, err error) {
+	ctx, span := otelTracer.Start(ctx, fmt.Sprintf("%s.ExecContext", otelName))
+	defer func() {
+		span.RecordError(err)
+		span.End()
+	}()
+
 	stmt, err := db.prepare(ctx, query)
 	if err != nil {
 		return nil, err
 	}
 	return stmt.ExecContext(ctx, args...)
 }
```

Comment:
```suggestion
func (db *DB) ExecContext(ctx context.Context, query string, args ...any) (result sql.Result, err error) {
	ctx, span := otelTracer.Start(ctx, fmt.Sprintf("%s.ExecContext", otelName))
	defer func() {
		span.RecordError(err)
		span.End()
	}()

	stmt, err := db.prepare(ctx, query)
	if err != nil {
		return nil, err
	}
	result, err = stmt.ExecContext(ctx, args...)
	return
}
```

Reply:
Keeping named returns as they are used in `span.RecordError(err)`, I would like to avoid using naked returns.